modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.json

```diff
@@ -0,0 +1,251 @@
+[
+	{
+		"availabilityEstimate": "",
+		"backOrders": true,
+		"categories": [
+			"Personal"
+		],
+		"description": "",
+		"displayAvailability": true,
+		"displayStockQuantity": true,
+		"externalReferenceCode": "RAYAP-001",
+		"height": 2,
+		"images": [
+		],
+		"length": 7,
+		"minStockQuantity": "",
+		"name": "Auto",
+		"price": "",
+		"promoPrice": "",
+		"shippable": false,
+		"sku": "RAYAP-001",
+		"specificationOptions": [
+		],
+		"taxCategory": "Normal Product",
+		"warehouse1": ""
+	},
+	{
+		"availabilityEstimate": "",
+		"backOrders": true,
+		"categories": [
+			"Personal"
+		],
+		"description": "",
+		"displayAvailability": true,
+		"displayStockQuantity": true,
+		"externalReferenceCode": "RAYAP-002",
+		"height": 2,
+		"images": [
+		],
+		"length": 7,
+		"minStockQuantity": "",
+		"name": "Home",
+		"price": "",
+		"promoPrice": "",
+		"shippable": false,
+		"sku": "RAYAP-002",
+		"specificationOptions": [
+		],
+		"taxCategory": "Normal Product",
+		"warehouse1": ""
+	},
+	{
+		"availabilityEstimate": "",
+		"backOrders": true,
+		"categories": [
+			"Personal"
+		],
+		"description": "",
+		"displayAvailability": true,
+		"displayStockQuantity": true,
+		"externalReferenceCode": "RAYAP-003",
+		"height": 2,
+		"images": [
+		],
+		"length": 7,
+		"minStockQuantity": "",
+		"name": "Property",
+		"price": "",
+		"promoPrice": "",
+		"shippable": false,
+		"sku": "RAYAP-003",
+		"specificationOptions": [
+		],
+		"taxCategory": "Normal Product",
+		"warehouse1": ""
+	},
+	{
+		"availabilityEstimate": "",
+		"backOrders": true,
+		"categories": [
+		],
+		"description": "",
+		"displayAvailability": true,
+		"displayStockQuantity": true,
+		"externalReferenceCode": "RAYAP-004",
+		"height": 2,
+		"images": [
+		],
+		"length": 7,
+		"minStockQuantity": "",
+		"name": "productEmpty",
+		"price": "",
+		"promoPrice": "",
+		"shippable": false,
+		"sku": "RAYAP-004",
+		"specificationOptions": [
+		],
+		"taxCategory": "Normal Product",
+		"warehouse1": ""
+	},
```

Comment:
Please, remove this product. It does not make sense to be added here. As a suggestion, you can talk with @gleicelisbino about this.

Reply:
Done !